OpenRA.Mods.D2k/Activities/DeliverUnit.cs

```diff
@@ -28,24 +30,25 @@ public class CarryUnit : Activity
 		readonly IPositionable positionable;
 		readonly IFacing cFacing; // Carryable facing
 		readonly IFacing sFacing; // Self facing
```

Comment:
Instead of having these comments I am fine with `selfFacing` and `carryableFacing`.